quantum/pointing_device/pointing_device_drivers.c

```diff
@@ -172,15 +170,30 @@ report_mouse_t cirque_pinnacle_get_report(report_mouse_t mouse_report) {
                 }
             }
         }
-#    endif
+#        endif
     }
 
-#    ifdef POINTING_DEVICE_GESTURES_CURSOR_GLIDE_ENABLE
+#        ifdef POINTING_DEVICE_GESTURES_CURSOR_GLIDE_ENABLE
 mouse_report_update:
-#    endif
+#        endif
     mouse_report.x = report_x;
     mouse_report.y = report_y;
+#    else
+    // Scale coordinates to arbitrary X, Y resolution
+    cirque_pinnacle_scale_data(&touchData, cirque_pinnacle_get_scale(), cirque_pinnacle_get_scale());
 
+    if (touchData.valid) {
+        mouse_report.buttons = touchData.buttons;
+#        ifdef MOUSE_EXTENDED_REPORT
+        mouse_report.x       = touchData.xDelta;
+        mouse_report.y       = touchData.yDelta;
+#        else
+        mouse_report.x = CONSTRAIN_HID_XY(touchData.xDelta);
+        mouse_report.x = CONSTRAIN_HID_XY(touchData.yDelta);
+#        endif
```

Comment:
```suggestion
        mouse_report.x = CONSTRAIN_HID_XY(touchData.xDelta);
        mouse_report.x = CONSTRAIN_HID_XY(touchData.yDelta);
```

Reply:
clang-format formats that the way I pushed it, I think that might raise some errors on checks if I commit it the way you propose.

Reply:
Formatting isn't the issue:

1. `#ifdef` isn't necessary because `CONSTRAIN_HID_XY()` covers both cases.
2. Typo assigning CONSTRAIN_HID_XY(touchData.`y`Delta) to mouse_report.`x`.

Reply:
Fixed